server/constants/constants.go

```diff
@@ -162,7 +162,7 @@ const (
 	PipelineRequestContextProjectName  = "projectName"
 	PipelineRequestContextRequestType  = "requestType"
 	PipelineRequestContextRequestName  = "requestName"
-	PipelineRequestContextProjectID    = "projectID"
+	PipelineRequestContextProjectID    = "projectId"
```

Comment:
why this change?

Reply:
To make it consistent with other constants